mitodl/open-discussions
# Ensure new users get the default NotificationSettings

#### What are the relevant tickets?
Fixes #541 

#### What's this PR do?
Adds a call to `ensure_notification_settings` when a user is created so they get the defaults.

#### How should this be manually tested?
Sign up for a new user in MM and navigate to OD, then visit `/api/v0/notification_settings/` and verify you get two entries back with the defaults.
